app/src/main/res/layout/help_fragment.xml

```diff
@@ -6,13 +6,12 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <TextView
-      android:id="@+id/welcome_text_view"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintLeft_toLeftOf="parent"
-      app:layout_constraintRight_toRightOf="parent"
-      app:layout_constraintTop_toTopOf="parent" />
+    <androidx.recyclerview.widget.RecyclerView
+      android:layout_height="match_parent"
```

Comment:
There are android based suggestions to make this file more consistent, please do that and following this same thing in other files too.

Reply:
@rt4914 Sorry sir, I am afraid that I got this point. Would you please describe this point in little detail?

Reply:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.